simple_network_sim/common.py

```diff
@@ -43,3 +45,22 @@ def __str__(self):
 
     def __repr__(self):
         return repr(self.f())
+
+
+class RepoInfo(NamedTuple):
+    git_sha: str
+    uri: str
+    is_dirty: bool
+
+
+def get_repo_info() -> RepoInfo:
+    try:
+        repo = git.Repo()
+    except git.InvalidGitRepositoryError:
+        return RepoInfo(git_sha="", uri="", is_dirty=True)
```

Comment:
I wonder if it would be better to hardcode the known URI rather than returning an empty string?